src/misc/ssh/ssh_test.c

```diff
@@ -0,0 +1,243 @@
+/* LibTomCrypt, modular cryptographic library -- Tom St Denis
```

Comment:
Please move `src/misc/ssh/ssh_test.c` to `tests/ssh_test.c` (+ do not forget to run `./updatemakes.sh`)